### DISCUSSION TITLE
[Unknown Mod][Game] Display a rumor in the Tavern when the Ultimate Artifact has already been found

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3391] [Ideas]
Display a rumor in the Tavern when the Ultimate Artifact has already been found.
The rumor appears randomly at the beginning of one of the four weeks following the week in which the final artifact was found.
This week (among the four possible weeks) is determined at the beginning of the scenario.

In fheroes2:

![Tavern rumor fh2 01](https://user-images.githubusercontent.com/43583994/117692350-3a4d1e80-b1bd-11eb-9199-ca1e640a81c4.png)

This is an option:
"game: display a rumor in the Tavern when the Ultimate Artifact has already been found"
This can be implemented directly in fheroes2.

**Comment by ihhub** [comment]
I think it'll be a bit discouraging as the player would know that someone else digged the artifact. What do you think?

**Reply by LeHerosInconnu** [reply]
@ihhub

It depends on the type of player (that's why I proposed it as an option).

For my part, I prefer to know that the Ultimate Artifact has been found and therefore it is no longer worth searching for it in vain.
It's a strategy game, information is important to adapt your strategy.
With this information, I can set up a plan to fight the opposing hero who holds the Ultimate Artifact, defeat him without him running away, and thereby make the Ultimate Artifact disappear.
